### ISSUE TITLE
Is there a way to avoid CORS block error on deployed implementations?

### ISSUE THREAD

**Issue by aimanamirul** [farshed/genius-lyrics-api#3]
**Current Behavior**

I have a React toy project which fetches current playback on Spotify and uses the song info to be put into the options parameter for getLyrics. I forked and modified the library where the access token is sent to the Genius API within the URL so the CORS block error won't happen on API requests. 

However, I'm now facing the [CORS block error on my Netlify deployment](https://prnt.sc/spt6c5) when the request is sent to a song lyric's page so I'm wondering if there's a way to solve this problem. Also, as can be seen in the image I also had modified the baseUrl in GetLyricsFromPath.js so that [thingproxy](https://github.com/Freeboard/thingproxy) is at front of the genius url, so it looks like below;

`const baseUrl = 'https://thingproxy.freeboard.io/fetch/https://www.genius.com';
`

Both the original baseUrl and modified baseUrl returns a CORS error on my deployed web app, however with the modified baseUrl it works well on localhost.

In a nutshell,
1. Using a proxy like thingproxy makes it work perfectly on localhost without using the Chrome disable web security flag method shown in the library readme, but not on production/deployment.
2. Without using the proxy it only works on localhost when Chrome runs with the disable web security flag being used and does not work on deployment.

**Steps to reproduce**

Deploy build of React web app using the library with Netlify/Surge/etc.

**Comment by farshed** [comment]
It seems that the structure of Genius webpages has been changed. I'm investigating the issue rn and will push a fix soon.

**Comment by farshed** [comment]
Hey, I just pushed a major release. Check it out and do tell me if it solves your problem.
https://github.com/farshed/genius-lyrics-api/releases/tag/v3.0.0

**Comment by farshed** [comment]
Closing this as the issue has likely been fixed. Feel free to reopen if the problem still persists
